Policies/github-terms-of-service.md

```diff
@@ -32,11 +32,11 @@ Thank you for using GitHub! We're happy you're here. Please read this Terms of S
 | [N. Disclaimer of Warranties](#n-disclaimer-of-warranties) | We provide our service as is, and we make no promises or guarantees about this service. **Please read this section carefully; you should understand what to expect.** |
 | [O. Limitation of Liability](#o-limitation-of-liability) | We will not be liable for damages or losses arising from your use or inability to use the service or otherwise arising under this agreement. **Please read this section carefully; it limits our obligations to you.** |
 | [P. Release and Indemnification](#p-release-and-indemnification) | You are fully responsible for your use of the service. |
-| [Q. Changes to these Terms of Service](#q-changes-to-these-terms) | We may modify this agreement, but we will give you 30 days' notice of changes that affect your rights. |
+| [Q. Changes to these Terms of Service](#q-changes-to-these-terms) | We may modify this agreement, but we will give you 30 days' notice of material changes. |
```

Comment:
Would a lawyer be willing to weigh in on the difference here?

For example:
- Which of these phrases is more clearly defined in the law?
- Which has broader scope?
- Why might Github have changed such a subtle wording?

Reply:
We are making this change so that we are consistent in describing this concept, rather than using two different terms that could suggest we mean something different. We did not use one term or the other with an intention that one is broader than the other.